### DISCUSSION TITLE
With version 14.0.7, infinispan-spring-boot3-starter-remote brings libraries incompatible with Spring Boot 3

### DISCUSSION THREAD

**Discussion by davidmestr** [infinispan/infinispan#10764] [General]
Adding `infinispan-spring-boot3-starter-remote` dependency:
```
<dependency>
      <groupId>org.infinispan</groupId>
      <artifactId>infinispan-spring-boot3-starter-remote</artifactId>
</dependency>
```

`infinispan-query-dsl` and `infinispan-remote-query-client` dependencies are added too, and both of them include `infinispan-commons`, but with Spring Boot 3 we need `infinispan-commons-jakarta`

We exclude `infinispan-commons`, but we don't known if it can be a problem.

**Comment by karesti** [comment]
@davidmestr infinispan-commons-jakarta is included with core-jakarta, excluding it is fine. however, if infinispan-commons is brought with query and remote query client, this will be fixed in the next 14 minor. opening a jira

**Reply by karesti** [reply]
https://issues.redhat.com/browse/ISPN-14730
